server/schema/resolvers.js

```diff
@@ -6,31 +6,21 @@ const resolvers = {
   Query: {
     // get single medicine
     medicine: async (parent, { medicineId }, context) => {
-      // delete after context implementation
-      return Medicine.findOne({ _id: medicineId });
-
       if (!context.user)
         throw new AuthenticationError('You need to be logged in!');
-      return Medicine.findOne({ _id: medicineId, userId: context.user.id });
+      return Medicine.findOne({ _id: medicineId, userId: context.user._id });
```

Comment:
nice! i hope it didn't give you too much grief to find these errors, i felt this in my soul